### ISSUE TITLE
[UX] move update settings under Configuration > Development

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#3388]
## Describe your issue or idea

I can never find the update manager settings, for the life of me. I always look in the development section of the menu first, followed by the system section. I never look for a tab on the reports pages. 

I recommend that we move `admin/reports/updates/settings` to `admin/config/development/updates` instead, with a `backdrop_goto_deprecated()`

**Comment by klonos** [comment]
We need to keep in mind that we seem to have consensus around:

- Merging all the "Update themes/modules/layouts" pages under a single page.
- Converting the "Available updates" page to a single "Update Manager" page, where the user can see a list of all installed themes/modules/layouts as well as Backdrop core + get an overview of the respective update status for all of these.
- Allow updating everything (selectively) from that single page.
- The settings page is to become the home for the new UI for enabling/configuring automatic security updates (#414)

Related issues: #3039 #2323 #2714

So if we are to move the settings page for this page, then we should also move this consolidated "Available updates"/"Update Manager" along with it.

...having said all that, I kinda like where that page is currently. If we were to move it someplace, then I think that I would like it to live directly under Configuration instead of another level deeper. So a `admin/config/updates/settings` or `admin/config/security/updates/settings` perhaps? Sorry, not sure how I feel about this yet.

**Comment by docwilmot** [comment]
> Merging all the "Update themes/modules/layouts" pages under a single page

@klonos agreed, but also remember https://github.com/backdrop/backdrop-issues/issues/1399#issuecomment-217969427 where we built the local tabs for Project Installer and agreed that whatever we do for Updates should ideally match Installer. Lets not forget to do those in tandem.

**Comment by jenlampton** [comment]
My intent here was to make the settings easier to find. because updates are development, I recommend `admin/config/development/updates`. 

> Converting the "Available updates" page to a single "Update Manager" page

These are two separate things. People (like me!) will still need the report page, even if they are managing the updates via Git, rather than via the update manager. I'll also still need to access the email settings.

I'm sticking to my recommendation :)

**Comment by klonos** [comment]
> because updates are development, I recommend `admin/config/development/updates`. I'm sticking to my recommendation :)

To me updates are also security, and performance/stability. I think that we should consider moving them under "Configuration" > "System" instead, along with "Security" and "Performance" (see #3624).

**Comment by klonos** [comment]
> ...because updates are development

I have come to the conclusion that I do not agree with this statement; especially when you see this from an end-user, non-dev point of view. As a site-builder, I may not touch code at all, but I still want to keep my site up to date, in order to keep it secure, fix problems, and add new features.

I think that "Configuration" > "System" > "Updates" > "Settings" may be more suitable.

**Comment by jenlampton** [comment]
I object to using `system` because that's sort of a cop-out "other" category, where we put things that we can't figure out where else to put. I never know what's in there, and I never look there for anything important.

> As a site-builder, I may not touch code at all

Just because you aren't touching code, doesn't mean you won't use anything in the "Development" section of the menu. My end-users clear the caches all the time. Many also manage configuration. Development doesn't mean "code". For non-coders this is even more the case because they may not have that correlation in mind.

This is the definition:
`The process in which someone or something grows or changes and becomes more advanced`

**Comment by klonos** [comment]
> I object to using `system` because that's sort of a cop-out "other" category...

Even better then; we could do "Configuration" > "Updates" > "Settings". That way, the users need not dig deep 😉

**Comment by jenlampton** [comment]
Better!

**Comment by avpaderno** [comment]
Yes, `/admin/config/system/updates/settings` would require the *Project Installer* module, since `/admin/config/system/updates` is a route that module defines. `admin/config/development/updates` would just require the _System_ module, which is always enabled, since `admin/config/development` is a route that module defines.

Changing the routes defined by the *Project Installer* and _System_ modules would make possible to have `/admin/config/system/updates` always available (because defined by the _System_ module). It would require adding a new route which is the default tab for `/admin/config/system/updates`, though.

**Comment by avpaderno** [comment]
It is also possible that the suggestion to use "Configuration" > "Updates" > "Settings" really means to use `/admin/config/updates/settings` as route, but `/admin/config/updates` is not an existing route.  
The *System* module could define `/admin/config/updates` as route for the update settings, though.
